models/basgra/R/run_BASGRA.R

```diff
@@ -426,6 +489,29 @@ run_BASGRA <- function(run_met, run_params, site_harvest, site_fertilize, start_
     # during the groowing season its depth will mainly be equal to the rooting depth, but during winter its depth will be ROOTD-Fdepth
     soilm <- output[thisyear, which(outputNames == "WAL")] # mm
     outlist[[length(outlist)+1]]  <- PEcAn.utils::ud_convert(soilm, "mm", "m") * 1000 # (kg m-3) density of water in soil
+
+    # WCL = WAL*0.001 / (ROOTD-Fdepth) Water concentration in non-frozen soil
+    # need to think about ice! but the sensors maybe don't measure that
+    ROOTD  <- output[thisyear, which(outputNames == "ROOTD")] 
+    Fdepth <- output[thisyear, which(outputNames == "Fdepth")] 
+    outlist[[length(outlist)+1]]  <- soilm * 0.001 / (ROOTD - Fdepth)
+    
+    # Additional C fluxes
+    outlist[[length(outlist)+1]] <- use PEcAn.utils::ud_convert(output[thisyear, outputNames == "FLITTC_LEAF"],
```

Comment:
```suggestion
    outlist[[length(outlist)+1]] <- PEcAn.utils::ud_convert(output[thisyear, outputNames == "FLITTC_LEAF"],
```
sorry that this crept in